.github/workflows/build_and_test.yml

```diff
@@ -55,34 +55,29 @@ jobs:
       if: contains(env.MODULES_TO_TEST, 'pytket-qsharp')
       run: dotnet tool install -g Microsoft.Quantum.IQSharp
 
-    - name: Set up Python 3.7
+    - name: Set up Python 3.8
       if: github.event_name == 'push'
       uses: actions/setup-python@v2
       with:
-        python-version: 3.7
-    - name: Build and test (3.7)
+        python-version: '3.8'
+    - name: Build and test (3.8)
       if: github.event_name == 'push'
       run: |
         ./.github/workflows/build-test nomypy
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
       if: github.event_name == 'pull_request'
       uses: actions/setup-python@v2
       with:
-        python-version: 3.8
-    - name: Build and test (3.8)
+        python-version: '3.9'
+    - name: Build and test (3.9)
       if: github.event_name == 'pull_request'
       run: |
         ./.github/workflows/build-test nomypy
-    - name: Set up Python 3.9
+    - name: Set up Python 3.10
       uses: actions/setup-python@v2
       with:
-        python-version: 3.9
-    - name: Build and test (3.9)
-      if: github.event_name == 'push' || github.event_name == 'pull_request'
-      run: |
-        ./.github/workflows/build-test nomypy
-    - name: Build, test and save (3.9)
-      if: github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel')
```

Comment:
what as happened to this release?

Reply:
Nothing, this is just a logical simplification of the workflow. The two steps, each conditionally executed, are combined into one, unconditionally executed.

Reply:
Perfect, Thank you for the explanation!